### PR TITLE
Feature/google search console

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Add google search console validation file
- Add basic robots txt

FRO-109 for linear link

### CODE REVIEW

**Review by diversemix — APPROVED** [review]
Woah neat!

File: frontend/public/google31c636b9724df5f5.html [hunk 1]
```diff
@@ -0,0 +1 @@
+google-site-verification: google31c636b9724df5f5.html
```
Comment (joel-wright) [inline frontend/public/google31c636b9724df5f5.html:1]: Just one question: does this ID apply to all our sites?

Comment (diversemix) [reply frontend/public/google31c636b9724df5f5.html:1]: Nope- just the URL https://app.climatepolicyradar.org ... we put the file on there to auth Google Search Console against this particular site

**Comment by PatFawbertMills** [comment]
Comment to link to linear: FRO-109 Generate an appropriate robots.txt file for different environments

File: frontend/public/google31c636b9724df5f5.html (hunk 1 above)
Comment (joel-wright) [reply frontend/public/google31c636b9724df5f5.html:1]: so, this is incorrect when deploying to dev then?

Comment (PatFawbertMills) [reply frontend/public/google31c636b9724df5f5.html:1]: Peter has set up the google search console to require this file in order to validate the site. It won't have any effect on any other environment.

**Pull request merged**